src/Bridge/Symfony/Bundle/Resources/views/DataCollector/api-platform.svg.twig

```diff
@@ -0,0 +1,53 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="{{ width|default(256.25) }}" height="{{ width|default(419.38) }}" viewBox="0 0 256.25 419.38">
```

Comment:
could we reference an online resource instead of adding this to the core? 

I'm not sure but it feels this file doesn't belong here.

Reply:
The Symfony documentation recommend importing svg via twig, that's why I did this like that

Reply:
okay keep it that way then :)